src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributor.java

```diff
@@ -51,74 +56,37 @@ public ActionDistributor(
   /**
    * Called on schedule to check for submitted actions then creates and distributes the actions to
    * action exporter or notify gateway
-   *
-   * @return the info for the health endpoint regarding the distribution just performed
    */
-  @Transactional
-  public DistributionInfo distribute() {
-    final DistributionInfo distInfo = new DistributionInfo();
+  public void distribute() {
```

Comment:
I wonder if this will have problems because there's no transaction - even read-only - which wraps the findall of the action types etc.